### PR TITLE
Update chapter-1.md

### PR DESCRIPTION
**Pull request by puffed-pescado**

### CODE REVIEW

File: pp/chapter-1.md [hunk 1]
```diff
@@ -48,3 +48,11 @@
...
 
 汪美华心里一惊：“坏了，我以前总听说过美国基础建设不如中国。我知道美国肯定没有我们华为的5G，但是连酒店都没有是我没想到的。我现在以及开始想念我亲爱的祖国了。”
+
+但是现在汪美华也没有别的去处了，他现在只好先接受妇女的建议。“谢谢你的好意，如果我能在你家住一晚，我会非常感激。”
+
+妇女友善地点了点头，然后领着汪美华走进她的家。这是一座小木屋，内部布置朴素而温馨。房间里有一张简单的床和一张桌子，墙上挂满了手绘的装饰画。
+
+“请坐吧。”妇女热情地邀请汪美华入座。她走向厨房，准备煮些茶水。
+
+汪美华环顾四周。房间的一角放置着一个小书架，上面摆放着几本古旧的书籍和手抄本。书籍的封面已经有些破旧，但依稀可以看见封面写着“HexSpell从入门到精通”。
```
Comment (nannoda) [inline pp/chapter-1.md:58]: I don't like the name of the book. Can you come up with something better?

**Review by nannoda — CHANGES_REQUESTED** [review]
Please see the message.

File: pp/chapter-1.md (hunk 1 above)
Comment (puffed-pescado) [reply pp/chapter-1.md]: Solved in commit 3f2661151a46f3d401e498e47cf28daf9536006b

**Review by nannoda — APPROVED** [review]
Looks good 😊

**Pull request merged**